app/backend/src/couchers/utils.py

```diff
@@ -49,11 +50,41 @@ def least_current_date():
     return datetime.now(timezone(timedelta(hours=-12))).strftime("%Y-%m-%d")
 
 
+# Note: be very careful with ordering of lat/lng!
+# In a lot of cases they come as (lng, lat), but us humans tend to use them from GPS as (lat, lng)...
+# When entering as EPSG4326, we also need it in (lng, lat)
```

Comment:
This smells like it was messing with you for a while :D

Reply:
Actually just very briefly, I was just plotting it and when London ended up off the east coast of Africa I knew it wasn't right! But it's really annoying unless you're completely on top of SRSs, etc and know all this stuff. Mostly they specify coordinates in (x,y), which is lng/lat, but 4326/GPS does it as lat/lng, because that's just what humans are used to. But then sometimes it's hard to tell whether these functions that do the conversions expect it lat/lng or lng/lat....